### ISSUE TITLE
Support import "module-name";

### ISSUE THREAD

**Issue by rousskov** [Mark-Simulacrum/attractifier#4]
Formatting sources containing an `import "foo";` line results in the following assertion:

```
../src/CodeGenerator.js:158
                throw new Error("AssertionError: " + message.replace(/\n/g, "\\n"));
                      ^
Error: tests/ProxyCase.js: AssertionError: currentValue should equal string: ""../src/misc/Lifetime"" == "{" at 9:7
    at CodeGenerator.assert (../src/CodeGenerator.js:158:23)
    at CodeGenerator.ensure (../src/CodeGenerator.js:490:18)
    at CodeGenerator._processSpecifiers (../../src/generators/modules.js:93:14)
    at CodeGenerator.ImportDeclaration (../../src/generators/modules.js:153:10)
    at CodeGenerator.print (../src/CodeGenerator.js:606:32)
    at ../../src/generators/index.js:17:14
    at arrayEach (/disks/protofroot/home/rousskov/Edit/pretty-generator/node_modules/lodash.foreach/node_modules/lodash._arrayeach/index.js:24:9)
    at /disks/protofroot/home/rousskov/Edit/pretty-generator/node_modules/lodash.foreach/index.js:25:9
    at CodeGenerator._statements (../../src/generators/index.js:16:5)
    at CodeGenerator.Program (../../src/generators/statements.js:6:10)
```

**Comment by Mark-Simulacrum** [comment]
Please confirm this is fixed in versions past v1.5.1. `npm update -g attractifier` to update attractifier given default install method.

**Comment by rousskov** [comment]
Confirmed as fixed.
